shippable.yml

```diff
@@ -0,0 +1,53 @@
+language: none
+
+
+build:
+  pre_ci_boot:
+    image_name: ansible/ansible
+    image_tag: centos7
+    pull: true
+    # we need a terminal or certain libc functions will fail
+    options: "-ti"
+  ci:
+    # debug info
+    - id
+    - export
+# prepare the tools
+    # this image is very bare and we need cron installed to test this role
+    - yum -y install cronie crontabs
+    # the Ansible package version is far too old (1.5.4)
+    - yum -y install python-pip python-devel make gcc gcc-c++ screen
+    - pip install ansible
+    # fix a bug using the cron module
+    - yum -y install wget
+    - wget -O /tmp/ansible_cron.patch https://github.com/ansible/ansible-modules-core/commit/531e09ce167c44387fc9c6505dd0453a7251b8f9.patch
+    - patch -d $(find /usr -wholename '*/system/cron.py' | xargs dirname | xargs dirname) -p1 </tmp/ansible_cron.patch
```

Comment:
Mhh, no the patch was sent to devel, so that's in 2.2. The backport to 2.1 happened to late. ANd as for why use --user, I do not know, I cut and paste a command from irc from a unrelated discussion. And yeah, --pre should be supported. That's outside of my access however, I guess we can open a ticket.